ui/app/services/auth.js

```diff
@@ -443,7 +443,7 @@ export default Service.extend({
   }),
 
   getOktaNumberChallengeAnswer(nonce, mount) {
-    let namespace = 'undefined';
+    let namespace = this.namespaceService.path;
```

Comment:
Namespace should already be attached to the ajax call automatically (see line 94 of this file) so we only need to pass it in if it's different than what's on the namespaceService currently. I *think* you should be able to remove this and the namespace from the options and it will work as expected

Reply:
Yes you're right I just tested it and it works.